src/collectors/rabbitmq/rabbitmq.py

```diff
@@ -144,7 +144,7 @@ def collect(self):
         self.collect_health()
         matchers = []
         if self.config['queues_ignored']:
-                for reg in self.config['queues_ignored']:
+                for reg in self.config['queues_ignored'].split():
```

Comment:
Do you want to use commas here instead of splitting on spaces? Do we use spaces elsewhere as a delimiter?